networkmanager/traffic.go

```diff
@@ -126,21 +134,26 @@ func newTrafficMonitor(trafficStorage TrafficStorage) (monitor *trafficMonitorin
 	return monitor, nil
 }
 
-/*******************************************************************************
+/***********************************************************************************************************************
  * Private
- ******************************************************************************/
+ **********************************************************************************************************************/
 
 func (monitor *trafficMonitoring) getTrafficChainBytes(chain string) (value uint64, err error) {
 	stats, err := monitor.iptables.ListWithCounters("filter", chain)
 	if err != nil {
-		return 0, err
+		return 0, aoserrors.Wrap(err)
 	}
 
 	if len(stats) > 0 {
 		items := strings.Fields(stats[len(stats)-1])
 		for i, item := range items {
 			if item == "-c" && len(items) >= i+3 {
-				return strconv.ParseUint(items[i+2], 10, 64)
+				// nolint:gomnd
+				if value, err = strconv.ParseUint(items[i+2], 10, 64); err != nil {
```

Comment:
Add 64 as exception to the linter settings instead of suppressing in the code.

Reply:
Done